media/vscode.css

```diff
@@ -1,20 +1,245 @@
 details > summary {
-  padding: 2px 6px;
-  border-bottom: 1px solid lightgray;
-  margin-bottom: 1%;
-  font-weight: bolder;
-  cursor: pointer;
-  font-size: large;
-}
-.comment-owner {
-  display: flex;
-  width: 100%;
-  justify-content: space-between;
-}
-.comment-date {
-  font-style: italic;
-  opacity: 0.5;
-}
-.comment-poster {
-  font-weight: bold;
-}
+    padding: 2px 6px;
+    border-bottom: 1px solid lightgray;
+    margin-bottom: 1%;
+    font-weight: bolder;
+    cursor: pointer;
+    font-size: large;
+  }
+  .comment-owner {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+  }
+  .comment-date {
+    font-style: italic;
+    opacity: 0.5;
+    color: lightgray;
+  }
+  .comment-poster {
+    font-weight: bold;
+    color: lightgray;
+  }
+  
+  /* Render markdown */
+  body {
+      color: lightgray;
+      font: 15px/1.3,sans-serif;
+  }
+  
+  a, a:visited {
+      color: #db6161;
```

Comment:
So we ARE going to overwrite the user settings?

Reply:
It's good to have our brand's colors but I'm open to getting a more detailed counter-opinion from you.